bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+require "cfme/versions"
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require "pry"
+# Pry.start
+
+require "irb"
+IRB.start(__FILE__)
```

Comment:
This file can literally not exist and you could just do this as a rake task.  There is only (was) one file in this entire project (by design), so there isn't anything fancy that you need a separate file for.

Also, based on how you setup the `.gemspec`, this gets brought along in the packaged gem, and is dead weight.